binCompatTest/src/main/scala-2/catsBC/MimaExceptions.scala

```diff
@@ -21,7 +21,8 @@
 
 package catsBC
 import cats.InjectK
-import cats.implicits._
+import cats.instances.all._
+import cats.syntax.all._
```

Comment:
Hmm.. That looks quite odd. As I understand, these two imports should not be required to happen simultaneously. So if they are required then perhaps some implicits are missing from the `cats.syntax.all._` path.

Reply:
> So if they are required then perhaps some implicits are missing from the

No, odd but correct. How can you resolve this implicit with only the syntax import?

https://github.com/typelevel/cats/blob/ccd576f0fb82af1747fe31fed28b833ff13cac03/binCompatTest/src/main/scala-2/catsBC/MimaExceptions.scala#L33

The syntax imports only work if you use the syntax :)

Reply:
Hmm... Not sure, what do you mean. This particular example does not require any additional imports to compile.
Because  `Monad` for `OptionT` is picked up from the `OptionT`'s companion, apparently.
Whereas `Monad[List]` is picked up from the `Invariant`'s companion automatically:
https://github.com/typelevel/cats/blob/ccd576f0fb82af1747fe31fed28b833ff13cac03/core/src/main/scala/cats/Invariant.scala#L138-L139

Reply:
In any case, I don't think it is an obstacle to get this PR merged :) Thank you for cleaning the imports up!

Reply:
Oh hm maybe I said a stupid :) and I was so sure I had it all figured out! let me take _one_ more look at this.

Reply:
Oh 🤦 it's because the whole point of this file, it to compile it against Cats v2.0.0, which pre-dates the re-organization that puts all implicits automatically in scope.

https://github.com/typelevel/cats/blob/ccd576f0fb82af1747fe31fed28b833ff13cac03/build.sbt#L265-L271

Ok, sorry, and thanks 😁

Reply:
I decided to revert to `cats.implicits` for this one. The point is to make sure we don't break compatibility with old code, so it seems better to write that import the old way.

Reply:
Yes, I think it makes sense, thanks!